migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc

```diff
@@ -26,6 +26,13 @@ include::modules/ossm-migrating-a-cluster-wide-deployment-using-the-istio-revisi
 
 include::modules/ossm-migrating-workloads-using-the-istio-revision-label.adoc[leveloffset=+2]
 
+include::modules/ossm-creating-a-default-revision-tag-and-relabeling-the-namespaces.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../migrating-gateways/ossm-migrating-gateways-assembly.adoc[Migrating gateways from Service Mesh 2 to Service Mesh 3]
+
 .Next steps
```

Comment:
At this point the migration is finished including the gateways. So next step here will be only to clean 2.6 installation.